cluster/gce/gci/configure-helper.sh

```diff
@@ -666,16 +667,16 @@ function start-kubelet {
     flags+=" --enable-debugging-handlers=false"
     flags+=" --hairpin-mode=none"
     if [[ "${REGISTER_MASTER_KUBELET:-false}" == "true" ]]; then
-      flags+=" --api-servers=https://${KUBELET_APISERVER}"
       flags+=" --register-schedulable=false"
       flags+=" --register-with-taints=node.alpha.kubernetes.io/ismaster=:NoSchedule"
+      flags+=" --kubeconfig=/var/lib/kubelet/kubeconfig"
```

Comment:
is there a reason that you put these flags in a different order than in the equivalent file in the container-linux directory? Since the files are largely the same, it'd be nice to keep them as similar as possible.